CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ All notable changes to this project will be documented in this file.
 * SUP-1072 Create new provider using framework plugin [[PR #286](https://github.com/buildkite/terraform-provider-buildkite/pull/286)] @jradtilbrook
 * SUP-1066 Migrate agent token to framework [[PR #289](https://github.com/buildkite/terraform-provider-buildkite/pull/289)] @jradtilbrook
 * Omit empty buildRetentionEnabled input [[PR #291](https://github.com/buildkite/terraform-provider-buildkite/pull/291)] @jradtilbrook
+* Consistent naming for environment variables [[PR #290](https://github.com/buildkite/terraform-provider-buildkite/pull/290)] @mcncl
+
+### Upcoming Changes
+[[PR #290](https://github.com/buildkite/terraform-provider-buildkite/pull/290)] changes the way that the organization environment variable is handled and brings
+it in line with the way it's used in Buildkite. Currently we check for `BUILDKITE_ORGANIZATION` but by changing it to `BUILDKITE_ORGANIZATION_SLUG` we can use it in the Terraform provider if a user opts to not set the `organization` input in their Terraform config file.
+
+`BUILDKITE_ORGANIZATION` will still be checked for backwards compatibility but will be removed in a future release.
```

Comment:
This should be under the "Unreleased" header above. v0.19.1 has already been released

Reply:
oops, perhaps the `Unreleased` header shouldn't be included in releases  in that case 🤔